### DISCUSSION TITLE
Running into some errors...

### DISCUSSION THREAD

**Discussion by Croestalker** [Extraltodeus/multi-subject-render#37] [Q&A]
Hey guys, I've been getting some errors... and I'd love to be able to fix these.  One of them is... "UnboundLocalError: local variable 'model' referenced before assignment", and another one is "rendering list index out of range."  Any help here would be great.  PS, I'm not as technically minded as I used to be, so simple steps please :0

I've also tried to look up what's going on but get no result :(

**Comment by gyattsoy** [comment]
I've got the same error but it seems to be caused by a different problem before that:
```
Traceback (most recent call last):
  File "C:\StableDiff\stable-diffusion-webui\extensions/multi-subject-render/scripts/simple_depthmap.py", line 105, in calculate_depth_map_for_waifus
    model = DPTDepthModel(path=model_path,backbone="swin2t16_256",non_negative=True,)
  File "C:\StableDiff\stable-diffusion-webui\repositories\midas\midas\dpt_depth.py", line 102, in __init__
    super().__init__(head, **kwargs)
  File "C:\StableDiff\stable-diffusion-webui\repositories\midas\midas\dpt_depth.py", line 55, in __init__
    hooks=hooks[backbone],
KeyError: 'swin2t16_256'
```
It always occurs after it has finished generating the foreground images.

**Reply by gyattsoy** [reply]
Ok I have done a ton of testing this past hour using the exact settings a guy in a different discussion shared.
I have no idea what exactly caused this issue but replacing the entire stable-diffusion-webui\repositories\midas folder with a new one downloaded here https://github.com/isl-org/MiDaS fixed it.
